### DISCUSSION TITLE
How to unlock servo after stopping

### DISCUSSION THREAD

**Discussion by mike-lloyd03** [ArminJo/ServoEasing#100] [Q&A]
When the servo first receives power, a user can manually rotate the shaft by hand. After a movement and then a command to stop, the servo is no longer free to move until power is removed.

Is there a way to free up the servo without cutting power to it?

**Comment by ArminJo** [comment]
Use detach() and reattach(). No servo signal is generated for a detached servo. Therefore, it is not blocked and can be moved manually.

**Reply by mike-lloyd03** [reply]
This doesn't seem to work. Detaching from the servo appears to keep it powered. Servo cannot manually move until cutting and restoring power to it.

```c++
  case CMD_STOP_MOTION: {
    println("Stopping");
    strncpy(status.name, "STOP", 16);
    servo.stop();
    servo.detach();
    break;
  }
```

**Reply by ArminJo** [reply]
Try the OneServo Example. It works for me 😀 .
